src/components/Book.js

```diff
@@ -1,22 +1,27 @@
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-const Book = ({ book }) => (
-  <li>
-    <div className="bookContainer">
-      <h2>{book.Title}</h2>
-      <p>{book.Author}</p>
-      <div className="actionsContainer">
-        <button type="button">like</button>
-        <button type="button">Comment</button>
-        <button type="button">delete</button>
+const Book = ({ book }) => {
+  const dispatch = useDispatch();
+  return (
+    <li>
+      <div className="bookContainer">
+        <h2>{book.Title}</h2>
+        <p>{book.Author}</p>
```

Comment:
When you save a book, you save it with the title and author properties in lowercase. However, here, you are trying to access them using uppercase letters. This is causing your component not to display any data. Please update this component so the book data is displayed👍